src/Resolver/Template.php

```diff
@@ -53,11 +53,11 @@ public function resolve($definition)
         $renderData     = [];
 
         if ($definition->has('provide')) {
-            foreach ($definition->get('provide') as $index => $definition) {
```

Comment:
Is this part of the PR functionality or just a little naming cleanup you did along the way?

Reply:
Yeah, two birds one PR to close #62 as well.